backend/src/apis/KlicktippController.ts

```diff
@@ -17,6 +17,9 @@ export const klicktippSignIn = async (
   firstName?: string,
   lastName?: string,
 ): Promise<boolean> => {
+  if (!CONFIG.KLICKTIPP) {
```

Comment:
As this is always the same, can we do something like:
```
export const callKlickTippAPI = (callback, args) = {
   if (!CONFIG.KLICKTIPP) {
      return true
    }
    return callback(args) 
}
````
I understand, that this is difficult in typescript

Reply:
Good point!

Reply:
Now is the question.. @ulfgebhardt is saying we should never use callback functions

Reply:
await and callback are different approaches to the same problem. Modern js uses await and callbacks only for constructs where it makes sense